common/djangoapps/edraak_i18n/helpers.py

```diff
@@ -1,7 +1,7 @@
 from django.utils.http import is_safe_url
 
 
-def get_any_safe_url(hosts, urls):
+def get_any_safe_url(hosts, urls, fallback):
```

Comment:
Nice it's only a fallback, why don't we give the argument a default value like `fallback ='/'`? This helps us reducing the code change later.